backend/src/services/complectations/complectations.service.ts

```diff
@@ -0,0 +1,225 @@
+import { prisma } from '@data/prisma-client';
+
+import type {
+  ModelReturnedData,
+  ComplectationReturnedData,
+  OptionType,
+  ComplectationsInput,
+  ModelReturnedDbType,
+  ComplectationReturnedDbType,
+} from '@autoline/shared/common/types/types';
+
+const getComplectationsById = async (
+  input: ComplectationsInput,
+): Promise<ModelReturnedData | ComplectationReturnedData> => {
+  const { userId, modelId, complectationId } = input;
+  const data = (await prisma.complectation.findUnique({
+    where: {
+      id: complectationId,
+    },
+    select: {
+      users_wishlists: {
+        where: {
+          user_id: userId,
+        },
+        select: {
+          id: true,
+          complectation: {
+            select: {
+              id: true,
+            },
+          },
+        },
+      },
```

Comment:
How wishlist impact getting of the complectation by its id? If we expect to return complectations only from the wishlist it should be expressed in the router and the name of the method

Reply:
We use this information to find out if this complectation is in the user's wish list in order to correctly display the like icon.